### ISSUE TITLE
项目运行错误

### ISSUE THREAD

**Issue by transtone** [yunsii/vite-react#9]
不支持较新的依赖更新了，只要把 `yarn.lock` 文件删除再运行，项目就报错。

**Comment by yunsii** [comment]
好歹贴个报错啊 \_(:з」∠)\_

**Comment by ianccy** [comment]
Hi, yunsii
Here is the error message snapshot.

![Screen Shot 2022-09-21 at 1 09 50 PM](https://user-images.githubusercontent.com/19364511/191419177-5c8b1d54-fdbb-427e-9291-9b7d11e7c6cf.png)

If i mark `dynamicImport.loading` and the problem will not happan.

- vite.config.ts
```
    vitApp({
      routes,
      // dynamicImport: {
      //   loading: './components/PageLoading',
      // },
      exportStatic: {},
    }),
```

**Comment by yunsii** [comment]
Thanks, I will try to resolve it tonight

**Comment by yunsii** [comment]
After a short research, I found that [vitjs/vit](https://github.com/vitjs/vit) example works very well. I will dive into the question in the weekend.

**Comment by yunsii** [comment]
@Ianpig Thanks for your feedback, I found the same error in https://github.com/vitjs/vit/issues/7 but the auther has no response then.

After all, the error casued by `use-subscription` the same major version of `^1.5.1`, I fixed it by `~1.5.1` in https://github.com/vitjs/vit/pull/9 so upgrade `vitjs/vit` & `vitjs/runtime` to lastest version will fix the error.

This tells me underlying framework should has more stricter version control.

**Comment by ianccy** [comment]
@yunsii thanks. i will try it later.
